### ISSUE TITLE
Pseudo-locales don't work in Blazor WebAssembly

### ISSUE THREAD

**Issue by uecasm** [dotnet/runtime#66498]

**Comment by yfranz** [comment]
I'm dealing the same problem. Is there any work around for it?

**Comment by ilonatommy** [comment]
@BrennanConroy, @sebastienros it seems like some issue connected with `Microsoft.Extensions.Localization`. What do you think?

**Comment by lewing** [comment]
@javiercn can you help us take a look at this

**Comment by javiercn** [comment]
@lewing I can try, but I'm not familiar with the area at all.

@ilonatommy do we see a delta between the behavior on .NET core vs .NET on wasm for `Microsoft.Extensions.Localization`?

**Comment by ilonatommy** [comment]
Yes, we do. For Blazor Server with asp NET core 3.1 runtime and Blazor Server with NET 7 runtime it's working fine. For Blazor Wasm with NET 7 runtime the behavior is as described in the issue. It's definitely something that Wasm is doing differently.
